pkg/fakerp/config.go

```diff
@@ -67,7 +67,7 @@ func GetPluginTemplate() (*pluginapi.Config, error) {
 	if err != nil {
 		return nil, err
 	}
-	genevaPullSecret, err := ioutil.ReadFile("secrets/acr-docker-pull-secert")
+	genevaPullSecret, err := ioutil.ReadFile("secrets/acr-docker-pull-secret")
```

Comment:
Why did this not break anything?

Reply:
file was misnamed in the secret itself - have duplicated the file to a new name, once this merges and goes back all the branches eventually we'll get rid of the old named file